migration-engine/connectors/migration-connector/src/steps.rs

```diff
@@ -260,6 +264,32 @@ impl MigrationExpression {
     }
 }
 
+#[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
+pub struct CreateCustomType {
+    pub custom_type: String,
+
+    pub r#type: String,
+    pub arity: ast::FieldArity,
```

Comment:
I think those fields are not enough yet. We must also capture the directives defined in the type alias. The bug report had a good example: `type CUID = String @id @default(cuid())`. This type alias would not work with this implementation.